e2e/cypress/integration/channel/more_channels_spec.js

```diff
@@ -41,10 +41,8 @@ describe('Channels', () => {
         cy.visit('/');
 
         // # Go to LHS and click "More..." under Public Channels group
-        cy.get('#sidebarChannelContainer').should('be.visible').within(() => {
-            cy.get('#publicChannelList').should('be.visible').within(() => {
-                cy.findByText('More...').scrollIntoView().should('be.visible').click();
-            });
+        cy.get('#publicChannelList').should('be.visible').within(() => {
```

Comment:
Look directly on `#publicChannelList` since `#sidebarChannelContainer` could be out of view.